### ISSUE TITLE
Binance pay support?

### ISSUE THREAD

**Issue by n0tMaster** [JKorf/Binance.Net#842]
Is the binance pay api integrated here?

**Comment by JKorf** [comment]
No, only the spot/futures APIs as described on https://binance-docs.github.io/apidocs/spot/en/#change-log are supported atm

**Comment by kayahasa** [comment]
any update here? Does still doesnt support binance pay?
